assets/js/modules/analytics-4/components/dashboard/ActivationBanner/SetupBanner.js

```diff
@@ -314,7 +322,10 @@ export default function SetupBanner( { onSubmitSuccess } ) {
 			</div>
 		);
 
-		if ( hasEditScope === false && ga4PropertyID === PROPERTY_CREATE ) {
+		if (
+			hasEditScope === false &&
+			( ga4PropertyID === PROPERTY_CREATE || ! getMeasurementID() )
```

Comment:
Thanks, @jimmymadon. Almost looks good to me, but let's use the current `measurementID` setting itself instead of calling the selector in the following condition.